tools/net/webserver/Python_Client.py

```diff
@@ -0,0 +1,358 @@
+		#print(chunkstr)
+		yield chunkstr.encode('utf-8')
+		time.sleep(0.000002)
+		i+=1
+
+	rem_data = total_data - ((i-1) * chunk_size)
+	chunkstr = hex(rem_data)[2:] + "\r\n"
+	if(len(str(i)) > rem_data) :
+		chunkstr += ''.join(random.choice(string.ascii_lowercase) for _ in range(rem_data)) + "\r\n"
+	else :
+		chunkstr += str(i) + ''.join(random.choice(string.ascii_lowercase) for _ in range(rem_data - len(str(i)))) + "\r\n"
+	#print(chunkstr)
+	yield chunkstr.encode('utf-8')
+	time.sleep(0.000002)
+	if is_trailer :
+		yield "0\r\nExpires: Fri, 21 Oct 2022 07:28:00 GMT\r\n\r\n".encode('utf-8')
+	else :
+		yield "0\r\n\r\n".encode('utf-8')
+
+
+def check_is_chunked_or_trailer(headers):
+	if is_trailer :
+		headers['TE'] = 'trailers'
+		headers['Trailer'] = 'Expires'
+		return
+	if(is_chunk):
+		headers['TE'] = 'chunked'
+
+
+def check_is_keep_alive(headers):
+	if(is_keep_alive):
+		headers['Connection'] = 'keep-alive'
+	else :
+		headers['Connection'] = 'close'
+
+
+#############  GET    ####################
+
+def get(total):
+	GET_P=0
+	GET_F=0
+	print("-------- GET test Start -------")
+	start_time = time.time()
+	for i in range(1,total+1):
+		print("--------Test {}/{}-------".format(i,total))
+		if(get_request()):
+			print("\nTest Case: Pass\n")
+			GET_P+=1
+		else :
+			print("\nTest Case: Fail\n")
+			GET_F+=1
+	print("---Test took %.6s seconds ---" % (time.time() - start_time))
+	print('keep-alive', is_keep_alive, ':: chunk', is_chunk, ' ::')
+	myTable = PrettyTable(["Test_Type", "Total_checks", "Pass", "Fail"])
+	myTable.add_row(["GET", total, GET_P, GET_F])
+	print(myTable)
+	return GET_P,GET_F
+
+
+def get_request():
+	headers = {'Accept':'application/json', 'user-agent': 'test_SRID', 'samsung':'Delhi'}
+	check_is_keep_alive(headers)
+	check_is_chunked_or_trailer(headers)
+	s = requests.session()
+	print("Headers Sent:")
+	for key,value in headers.items():
+		print(key+" : "+value)
+	P, F = 0, 0
+	for i in range(keep_alive_iter):
+		try:
+			r = s.get(url, headers=headers, verify=cert)
+			if(r.status_code==200):
+				if(is_keep_alive and (r.headers.get('Connection')=='close' or r.headers.get('Connection')=='close, close')):
+					F += 1
+					continue
+				print("\nResponse Status : "+str(r.status_code)+ "\nResponse Phrase : " +str(r.reason))
+				for key,value in r.headers.items():
+					print(key+" : "+value)
+				print("Content received : "+r.text)
+				P += 1
+			else :
+				F += 1
+			r.raise_for_status()
+		except requests.exceptions.RequestException as e:
+			print(e)
+			F += 1
+	s.close()
+	if(P == keep_alive_iter):
+		return True
+	return False
+
+
+####################### POST ##########################
+
+def post(total):
+	POST_P=0
+	POST_F=0
+	print("-------- POST test Start -------")
+	start_time = time.time()
+	for i in range(1,total+1):
+		print("--------Test {}/{}-------".format(i,total))
+		if(post_request()):
```

Comment:
gap between 2 requests may be required, 
can we take this value as input and use when provided by user